### DISCUSSION TITLE
Two shaft conection

### DISCUSSION THREAD

**Discussion by augustodc76** [petrobras/ross#1036] [Q&A]
Hi ROSS developers
I did some trials with ROSS some months ago with a very good results, my congratulations to the ROSS development team!.
Now I would like to evaluate a special case, I have 2 coaxial shafts and I´m having issues with:
1.  Im not able to properly locate axialy the 2nd shaft (orange vs blue shaft), the parameter "shaft_start_pos"  is not recognized.
2.  I cant find an "element" to conect both shafts like a "flange" between (green element)
Do you have any recomendation to model this case?
Thanks in advance

**Comment by ViniciusTxc3** [comment]
Hi @augustodc76 , I'd not fully understand the your question.
To join two shafts it is possible to use a disk. As in the following example:

```python
import ross as rs

Q_ = rs.Q_
steel = rs.Material("steel", rho=7800, E=200e9, Poisson=0.27)
L = 0.2
idl = 0.03
odl = 0.08
N = 6
shaft = [rs.ShaftElement(L=L, idl=idl, odl=odl, material=steel) for i in range(N)]
disks = [
    rs.DiskElement.from_geometry(n=3, material=steel, width=0.08, o_d=0.4, i_d=0.08)
]

rotor1 = rs.Rotor(shaft_elements=shaft, disk_elements=disks)
rotor1.plot_rotor().show()
```

This code was adapted from the documentation [Example 16](https://ross.readthedocs.io/en/stable/user_guide/example_16.html?highlight=coaxial%20shaft).
